packages/e2e-tests/tests/playwright-01_basic-panel.test.ts

```diff
@@ -0,0 +1,114 @@
+import { openDevToolsTab, startTest } from "../helpers";
+import {
+  getSelectedTestCase,
+  getTestCaseSteps,
+  getTestRecordingBackButton,
+  getTestRecordingTrees,
+  getTestRowChevron,
+  getTestRows,
+  getTestSections,
+  getTestSuiteBranch,
+  getTestSuiteDate,
+  getTestSuiteDuration,
+  getTestSuitePanel,
+  getTestSuiteResultsFailedCount,
+  getTestSuiteResultsPassedCount,
+  getTestSuiteResultsSkippedCount,
+  getTestSuiteUser,
+  openCypressTestPanel,
+} from "../helpers/testsuites";
+import { waitFor } from "../helpers/utils";
+import test, { expect } from "../testFixtureCloneRecording";
+
+test.use({ exampleKey: "playwright/breakpoints-05" });
+
+test("playwright-01: Basic Test Suites panel functionality", async ({
+  pageWithMeta: { page, recordingId },
+  exampleKey,
+}) => {
+  await startTest(page, recordingId);
+  await openDevToolsTab(page);
+
+  await openCypressTestPanel(page);
+
+  const testPanel = getTestSuitePanel(page);
+
+  const isVisible = await testPanel.isVisible();
+  expect(isVisible).toBe(true);
+
+  // These are nested, but at least one should exist
+  // on the test suites panel
+  let initialRecordingTreesCount = 0;
+  const initialRecordingTrees = getTestRecordingTrees(page);
+  await waitFor(async () => {
+    initialRecordingTreesCount = await initialRecordingTrees.count();
+    expect(initialRecordingTreesCount).toBeGreaterThanOrEqual(1);
+  });
+
+  // has 1 test
+  const rows = getTestRows(page);
+  await waitFor(async () => {
+    await expect(rows).toHaveCount(1);
+  });
+
+  const firstTest = rows.first();
+
+  // displays the nav chevron on hover
+  const chevron = getTestRowChevron(firstTest);
+  await chevron.isHidden();
+  await firstTest.hover();
+  await chevron.isVisible();
```

Comment:
I guess you're trying to wait until `chevron` is visible but that's not what this line does. You need to use `await expect(chevron).toBeVisible();`. The same goes for `await chevron.isHidden();` at line 58.

Reply:
:+1: I'll point out that this test was lifted directly from the cypress tests so those must be wrong too.